test/usecases/store_auth_token_use_case_test.dart

```diff
@@ -9,10 +9,15 @@ void main() {
   group('StoreAuthTokenUseCaseTest', () {
     late StoreAuthTokenUseCase storeAuthTokenUseCase;
     late MockSecureStorage mockSecureStorage;
+    late MockSharedPreferencesStorage mockSharedPreferencesStorage;
 
     setUp(() async {
       mockSecureStorage = MockSecureStorage();
-      storeAuthTokenUseCase = StoreAuthTokenUseCase(mockSecureStorage);
+      mockSharedPreferencesStorage = MockSharedPreferencesStorage();
+      storeAuthTokenUseCase = StoreAuthTokenUseCase(
+        mockSecureStorage,
+        mockSharedPreferencesStorage,
+      );
```

Comment:
Shall we update the test to verify `_sharedPreferencesStorage.saveLoginStatus(true);` was called? 💪

Reply:
@Tuubz in `StoreAuthTokenUseCaseTest` we already check 2 case success and failure which contains the execution of `_sharedPreferencesStorage.saveLoginStatus(true);`, so at this point I think we don't need to check again.What do you think about it?